reccmp/isledecomp/compare/core.py

```diff
@@ -67,12 +68,16 @@ def lookup(addr: int, size: int) -> Optional[bytes]:
 class Compare:
     # pylint: disable=too-many-instance-attributes
     def __init__(
-        self, orig_bin: IsleBin, recomp_bin: IsleBin, pdb_file: str, code_dir: str
+        self,
+        orig_bin: IsleBin,
+        recomp_bin: IsleBin,
+        pdb_file: Path | str,
```

Comment:
Wouldn't making them `Path` throughout the code make more sense? It's a form of intent and self-documenting code.

Reply:
`argparse` must support creating a `Path` object from the command line, right? If so, then it should be `Path` all the way down. But we could leave `Path | str` for now since this is not the only lingering type issue.

Reply:
Yes, simply add `type=Path`
Keeping `Path | str` is fine with me :)